examples/ibm-cloudant-database/modules/config-database/versions.tf

```diff
@@ -0,0 +1,27 @@
+#####################################################
+# Cloudant
+# Copyright 2021 IBM
+#####################################################
+
+/***************************************************
+NOTE: To source a particular version of IBM terraform provider, configure the parameter `version` as follows
+terraform {
+  required_version = ">=0.13"
+  required_providers {
+    ibm = {
+      source = "IBM-Cloud/ibm"
+      version = "1.21.0"
+    }
+  }
+}
+If we dont configure the version parameter, it fetches the latest provider version.
+****************************************************/
+
+terraform {
+  required_version = ">=0.13"
```

Comment:
I note that in some places the version is `0.12`, I guess `0.13` is needed for modules, but only `0.12` for the provider.

Reply:
This is one thing that made working with this PR that hard - I have no idea why it is suddenly 0.13 here of all places. It might be intentional or it might be just a random noise - with original author been both knowledgeable and kind of ... relaxed on consistency, it's just impossible to guess, I changed other things like that and in some cases that was just fine and in others I had to go back and revert them, because they subtly didn't work.

In this case I dropped it to 0.12 and run a complete manual test and haven't noticed any degradation, so I'm dropping those to 0.12 for consistency with the rest of `versions.tf` across project examples.